aspnetcore/web-api/http-repl-tool.md

```diff
@@ -0,0 +1,338 @@
+---
+title: Use the HTTP REPL tool
+author: scottaddie
+description: Learn how to use the HTTP REPL .NET Core Global Tool to browse and test an ASP.NET Core web API.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: scaddie
+ms.custom: mvc
+ms.date: 06/21/2019
+uid: web-api/http-repl-tool
+---
+# Use the HTTP REPL tool
+
+The HTTP REPL is a cross-platform tool that's supported everywhere .NET Core is supported. It is a light weight command line tool for making HTTP calls to test APIs and view their results.
+
+## Prerequisites
+
+* [.NET Core SDK 3.0.100](https://dotnet.microsoft.com/download/dotnet-core/3.0) or later
+
+## Installation
+
+To install the HTTP REPL:
+
+```console
+dotnet tool install -g dotnet-httprepl

```

Comment:
Do we need to include the version number prior to it landing in the non-pre-release state? could be helpful prior to release.